yaptide/persistence/models.py

```diff
@@ -269,6 +269,12 @@ class CeleryTaskModel(TaskModel):
     __tablename__ = 'CeleryTask'
     id: Column[int] = db.Column(db.Integer, db.ForeignKey('Task.id', ondelete="CASCADE"), primary_key=True)
     celery_id: Column[str] = db.Column(db.String, nullable=False, default="", doc="Celery task ID")
+    sim_pid: Column[int] = db.Column(db.Integer,
+                                     nullable=True,
+                                     doc='Id of simulation process used to communicate with shieldhit process')
```

Comment:
we typically speak of PID, that would clearly differentiate it from celery IDs.

Reply:
does it mean you introduce a column which is used only for SHIELD-HIT12A ? if that is the case, write in doc what is its meaning for Fluka ?

Reply:
Celery id is a id of a celery worker, sim_pid is a PID of a shieldhit process